src/content/masking.md

```diff
@@ -0,0 +1,28 @@
+---
+slug: '/template'
```

Comment:
rename this to `masking` or into something fun (`james bond`)

Reply:
done